bin/shutter

```diff
@@ -7831,8 +7831,9 @@ if ($app->is_running) {
 
 			my $filename_template = quotemeta $filename_value;
 
-			#replace %NNN by a regex to search for digits
-			$filename_template =~ s/\\\%N+/(\d+)/g;
+			#replace %NNN by a \d+ regex to search for digits
+			#also take into account conflicted filenames Ex.: "_014(002)"
```

Comment:
yes, because in the previous pattern, _018.jpg would be matched with "18" kept for counter, but _018(002).jpg would have been ignored because "(" ")" are not parts of "\d".
Now, even if _018.jpg does not exist but _018(002).jpg exists, the last previous increment will still be "18".

FYI, I did think about that before the first commit, but I spitted it in 2 commits so that the behavior difference is explicit.

Reply:
Ah, got it. But I think the pattern needs to be more generic. I don't remember ever seeing files like _018(002).jpg but I saw somewhere with a space before `(`.

Also, please use `(?:...)` for the non-capturing group